static/resources/harness.js

```diff
@@ -381,7 +383,7 @@
   }
 
   function runWindow(callback) {
-    setCurrentExposure('Window');
+    setCurrentExposure('Window', !!pending.Window);
```

Comment:
Can we just not call `setCurrentExposure` if there aren't any pending tests, by moving this call into the if statement?

Reply:
I tried that at first but it ended up causing other issues.  The code that catches if an exposure set is run twice depends on the exposure sets' run functions to be called in that order, and if we skip one, we end up with the tests just freezing (since the collector only lets the first completion of an exposure set continue on).

Reply:
If the problem is with `navigator.mediaDevices.getUserMedia()` returning the same promise multiple times or if the `.then()` callback of certain promises gets called multiple times, is there a more local workaround for those odd situations that doesn't affect the overall harness?